### ISSUE TITLE
Not able to generate the PDF from HTML

### ISSUE THREAD

**Issue by Jaimin-Devhero** [mpdf/mpdf.github.io#180]
> PHP: 8.0.19

I am generating the PDF from the HTML, but it's not generating for some reason. 

**Below is my code:**

`$html = '<table style="padding-right: 20px; padding-left: 20px; font-size:14px;font-family: helvetica"><tr><td style="height:560px; "><p>I hope you are getting well Reesha <3 You’ve been holding shit down since day1 gang I need you aliveeee buh hopefully you eating now too or imma be mad Luhhh youuuu. **<<<3333** Adam Alexander Mlamali</p></td><tr></table>';
$mpdf->WriteFixedPosHTML($html, 105, 0, 104, 150, 'auto');
$mpdf->Output("", "I");`

What I have figured out is that if I remove the "<<<" from the HTML its works fine. Can someone guide me whats the issue with <<< here and how I can generate the PDF without removing the "<<<"?

**Comment by Jaimin-Devhero** [comment]
Hello @finwe 

I am still facing this issue. can you guide me?

**Comment by finwe** [comment]
No. Read contributing guidelines you deleted when you created the issue.

**Comment by Jaimin-Devhero** [comment]
Hello @finwe 

I have read the guideline and explained my problem properly with the code I am using.

So you can understand the bug properly.
